### DISCUSSION TITLE
Convert html to tiptap format

### DISCUSSION THREAD

**Discussion by svendehens** [awcodes/filament-tiptap-editor#435] [Q&A]
Is there a way to convert existing html to tiptap json? I'd like to migrate an app that used Filament's Richeditor to the tiptap editor.. Thanks for the reply!

**Comment by awcodes** [comment]
There's no guarantee that it will be a 100% accurate conversion coming from a different editor, but you can pass the html content into `tiptap_converter()->asJSON($html)` to get a JSON version of it.
